js/common.js

```diff
@@ -0,0 +1,8 @@
+const $ = (select, el = document) => el.querySelector(select);
+
+const SIGN_INFO_REGEX = {
+  email: /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/,
+  password: /^[A-Za-z0-9]{8,15}$/,
+};
+
+export { $, SIGN_INFO_REGEX };
```

Comment:
export를 변수를 선언할때 해줄건지, 하단에 몰아서 해줄건지는 취향의 영역이라고 생각하는데요~

개인적으로는 아래와 같이 더 많이 하는 것 같기는 해요! 팀의 컨벤션에 맞게만 사용되면 상관없을 것 같아요!

```suggestion
export const $ = (select, el = document) => el.querySelector(select);

export const SIGN_INFO_REGEX = {
  email: /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/,
  password: /^[A-Za-z0-9]{8,15}$/,
};
```